Langbot新版本适配

langbot更新之后不再使用以前的config，会导致waifu拉取用户信息出错，请求进行适配

是这个报错吗亲爱的
manager.py (241) - [ERROR] : 插件 Waifu 处理事件 PersonMessageReceived 时发生错误: SessionManager.get_conversation() missing 2 required positional arguments: 'pipeline_uuid' and 'bot_uuid'

> 是这个报错吗亲爱的 manager.py (241) - [ERROR] : 插件 Waifu 处理事件 PersonMessageReceived 时发生错误: SessionManager.get_conversation() missing 2 required positional arguments: 'pipeline_uuid' and 'bot_uuid'

是的，langbot更新导致的

--- a/main.py
+++ b/main.py
@@ -731,7 +731,7 @@ class WaifuPlugin(BasePlugin):
         else:
             self.ap.logger.warning("query.pipeline_config is None, using empty prompt for get_conversation.")
 
-        conversation = await self.ap.sess_mgr.get_conversation(query, session, prompt_config_from_pipeline)
+        conversation = await self.ap.sess_mgr.get_conversation(query, session, prompt_config_from_pipeline,query.pipeline_uuid,query.bot_uuid)
 
         use_model = conversation.use_llm_model # Changed from conversation.use_model
 

